src/main/kotlin/org/jetbrains/research/testspark/display/generatedTestsTab/GeneratedTestsTabBuilder.kt

```diff
@@ -209,7 +207,7 @@ class GeneratedTestsTabBuilder(
     private fun applyTests() {
         // Filter the selected test cases
         val selectedTestCasePanels =
-            generatedTestsTabData.testCasePanels.filter { (it.value.getComponent(0) as JCheckBox).isSelected }
+            generatedTestsTabData.testCaseNameToPanels.filter { (it.value.getComponent(0) as JCheckBox).isSelected }
```

Comment:
What component resides under the 0th index in `getComponent(0)`? It is better to comment on it or, even better, move the `0` into a named constant.

Reply:
@arksap2002 please check this out